package.json

```diff
@@ -53,16 +53,16 @@
     "@storybook/addons": "^3.4.8",
     "@storybook/react": "^3.4.8",
     "babel-core": "^6.26.3",
-    "babel-eslint": "^8.2.2",
+    "babel-eslint": "9.0.0",
     "babel-preset-env": "^1.7.0",
     "babel-preset-react-app": "^3.1.2",
     "babel-register": "^6.26.0",
     "babel-runtime": "^6.26.0",
     "chromedriver": "^2.40.0",
     "enzyme": "^3.3.0",
     "enzyme-adapter-react-16": "^1.1.1",
-    "eslint": "^4.9.0",
-    "eslint-config-airbnb": "^16.1.0",
+    "eslint": "5.6.0",
```

Comment:
Note: CRA requires to update eslint, which requires us to update eslint-config-airbnb which enables additional eslint rules. Some of them I disabled above